src/Install/Steps/EnableBundledExtensions.php

```diff
@@ -109,7 +115,7 @@ private function loadExtensions()
 
                 return $extension;
             })->filter(function (Extension $extension) {
-                return in_array($extension->getId(), self::EXTENSION_WHITELIST);
+                return in_array($extension->getId(), $this->enabledExtensions);
```

Comment:
Since we're now using the constant in the constructor, could we please move the declaration of the constant to the top of the class ?